examples/override-example/override.json

```diff
@@ -8,8 +8,46 @@
     "use_appid": false,
     "use_data": false
   },
-  "clusters": [],
-  "enable_transit_gateway": true,
+  "clusters": [
```

Comment:
you should update this example readme since it now looks like it will deploy a cluster